app/javascript/packs/users_viewing_page.js

```diff
@@ -32,6 +32,13 @@ const subscription = consumer.subscriptions.create({ channel: 'ContactChannel',
 });
 
 
+const cleanup = () => {
```

Comment:
it must have disappeared in the merge

Reply:
Can we remove the `console.log()` before merging?

Reply:
sure